### DISCUSSION TITLE
Welcome to hc3-to-mqtt Discussions!

### DISCUSSION THREAD

**Discussion by alexander-vitishchenko** [alexander-vitishchenko/hc3-to-mqtt#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarcoAVoegele** [comment]
Hi Alexander,
thank you for the HC3 integration into HA..
I have a couple of devices, that do not appear as entities in HA.
Like fibaro power plugs or fibaro heat controll...

Is there a way to integrate them as well.

Thank you so much,

Marco

**Reply by alexander-vitishchenko** [reply]
Hi @MarcoAVoegele,

This might be possible and I need to have more info about your devices submitted with https://github.com/alexander-vitishchenko/hc3-to-mqtt/issues/new?assignees=&labels=&template=feature_request.md&title=

**Comment by jaspweowaale** [comment]
Hi 

I use your code but im in a location that I can't use the " donate"  function ( Vietnam ) are you able to share you Paypal to me directly I like to donate to you 

I was wondering if its possible also to import MQTT info and not just export, Im asking as Im running both HA and Fibaro at the same time

**Comment by alexander-vitishchenko** [comment]
Hi @jaspweowaale,

This QuickApp exports Fibaro devices to Home Assistant. If I understand your question correctly, you are asking whether it is possible to export Home Assistant devices to Fibaro? Theoretically, this could be an option, but it would require creating an entirely new QuickApp. This might be worth considering if there is significant demand and support, though it currently seems to be a niche use case.

I've updated the PayPal settings, feel free to try.

**Comment by jaspweowaale** [comment]
Hope all is well,

Yeah, my idea after seeing the export from Fibaro was that having an import
function also would be beneficial, thinking that 80% of the work is done in
the export.

But I also decided to leave Fibaro. The brand looks to be mostly dead, and
support does not answer or offer any help. The one time in 8 years I needed
help, I got a cut-and-paste answer. The forums are overrun with fanboys,
locked into the "status" they seem to have achieved. I have seen this more
than once, as I have a few accounts. Some of them look like I am new to the
subject and haven't been working with this for years. One thing that stands
out is how rude and impolite they are to anybody asking anything that is
not 100% Fibaro-related.

What made me decide to review my Fibaro use was one thread about Zigbee. It
would be a winning move to add it, but we were told in 2022 it would be
done by Christmas. It was not. Somebody asked about a device and function
was getting so much hate that I decided to get a Pi5 and set up a Zigbee
network on HA.

I have mostly moved all my energy-related things, but as you might have
guessed, after 8 years I have a lot of Fibaro stuff. So, after thinking, I
got myself a Z-wave USB for HA. That fixed my import problem. I liked the
dashboards in Fibaro, but I am working on HA to make something that covers
my needs.

So, the HC3, HC2, and Light will be offline soon. I checked your PayPal
thing, but I don't think my idea will be worth spending time on. I see a
few of bugs in the current code like this. My system is all in English.

![image](https://github.com/alexander-vitishchenko/hc3-to-mqtt/assets/66903625/6b52b938-87bd-45db-90c5-d96293b1fd20)


Jasper

**Reply by alexander-vitishchenko** [reply]
Hi @jaspweowaale in case your interest in Fibaro is primarily driven by Zigbee, have you considered using Home Assistant software and hardware ecosystem? This might be a simpler approach comparing to creating a new QuickApp with a little reuse possible from the current QuickApp.

**Reply by jaspweowaale** [reply]
Yah, as I was saying above,  I made that decision after a lack of Support and after seeing the forums being so hostile  to new ideas, 

Send me a direct link to Paypal to qcao24-99@matrongvang.com, I use your code, and like to support you

**Comment by jaspweowaale** [comment]
Yah,

The problem with PayPal is the same,  Im not in a location where Paypal allow me
to donate, Im able to send you funds but NOT donations

![image](https://github.com/alexander-vitishchenko/hc3-to-mqtt/assets/66903625/ae84cb2d-70e7-4ca1-a8c4-892c2b89a5e4)

**Comment by sakkhho** [comment]
Hello @alexander-vitishchenko 
i'm using (and i want to keep) my HC3 for different reason.
However there are couple of device such my HeatPump Vaillant that i'm not able to manage through HC3 but HA can.
https://community.home-assistant.io/t/myvaillant-integration/542610

i was wondering if there is a possibility to control HA from HC3 (and import Device from HA)

regards
